### ISSUE TITLE
right prompt color error

### ISSUE THREAD

**Issue by DuncanHuang** [bhilburn/powerlevel9k#955]
Hi, I'm setting "POWERLEVEL9K_RIGHT_PROMPT_ELEMENTS" using "ip", but color error

![2018-08-16_10-40-27](https://user-images.githubusercontent.com/1020091/44184581-4c9bea00-a142-11e8-85c6-175ec9a1e699.png)

I find is 264897a426c7286d0734895b35ea437a6accc1bc this commit after, i checkout to 87acc51acab3ed4fd33cda2386abed6f98c80720 this problem does not exist.

![2018-08-16_10-41-02](https://user-images.githubusercontent.com/1020091/44185119-085e1900-a145-11e8-99d4-981ce993c618.png)

I try modify file "functions/colors.zsh", change new version

```text
'black'   '000'
'red'     '001'
'green'   '002'
'yellow'  '003'
'blue'    '004'
'magenta' '005'
'cyan'    '006'
'white'   '007'
```
to old version

```text
'black'  '000'
'maroon' '001'
'green'  '002'
'olive'  '003'
'navy'   '004'
'purple' '005'
'teal'   '006'
'silver' '007'
```
this problem does not exist, i think this may be the color mapping error.

This problem in macOS does not exist, but in CentOS 6、7 is exist.

Provide my setting parameters

```text
ZSH_THEME="powerlevel9k/powerlevel9k"
POWERLEVEL9K_CONTEXT_TEMPLATE="%n"
POWERLEVEL9K_LEFT_PROMPT_ELEMENTS=(context dir vcs)
POWERLEVEL9K_RIGHT_PROMPT_ELEMENTS=(ip root_indicator background_jobs command_execution_time time)
POWERLEVEL9K_SHORTEN_DIR_LENGTH=3
POWERLEVEL9K_SHORTEN_STRATEGY=truncate_from_right
POWERLEVEL9K_PROMPT_ON_NEWLINE=true
POWERLEVEL9K_PROMPT_ADD_NEWLINE=true
POWERLEVEL9K_MULTILINE_FIRST_PROMPT_PREFIX='╭'
POWERLEVEL9K_MULTILINE_LAST_PROMPT_PREFIX='╰'$'\uE0B0 '
POWERLEVEL9K_EXECUTION_TIME_ICON=''
POWERLEVEL9K_COMMAND_EXECUTION_TIME_THRESHOLD=0
POWERLEVEL9K_COMMAND_EXECUTION_TIME_PRECISION=4
```

**Comment by dritter** [comment]
Hi @DuncanHuang !

Thanks for the bug report. You are right, some colors gone missing in the last release. We fixed that in #929. Could you try out #944 , which is our preparation branch for the upcoming release?

**Comment by DuncanHuang** [comment]
#944 fixed this problem, but the color is different from before

![2018-08-16_14-08-22](https://user-images.githubusercontent.com/1020091/44191466-2ab06080-a15e-11e8-8146-fc96208f6f1f.png)

**Comment by dritter** [comment]
Hi @DuncanHuang , I created #959 to discuss the color issue.

**Comment by bhilburn** [comment]
Fixed by merge of #959!
